mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb

```diff
@@ -17,13 +17,15 @@ func TestAccContainerCluster_basic(t *testing.T) {
 	t.Parallel()
 
 	clusterName := fmt.Sprintf("tf-test-cluster-%s", acctest.RandString(t, 10))
+	networkName := acctest.BootstrapSharedTestNetwork(t, "gke-cluster", false)
+	subnetworkName := acctest.BootstrapSubnet(t, "gke-cluster", networkName)
```

Comment:
The default subnetwork allocates a /20 (4096 IPs), but this allocates a /28 (16 IPs). We'll want to expand the size of the range we provision!